resources/js/components/Common/PasswordInput.vue

```diff
@@ -0,0 +1,16 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import CustomInput from './CustomInput.vue'
+import EyeDynamicIcon from '../Icons/EyeDynamicIcon.vue'
+defineProps<{ error?: string }>()
```

Comment:
No extra line after imports

Reply:
![image](https://github.com/user-attachments/assets/453973c3-5552-44b3-b074-bf5b9c0f5b76)

Why does it not get outdated?